torchvision/prototype/transforms/functional/_temporal.py

```diff
@@ -0,0 +1,24 @@
+import torch
+
+from torchvision.prototype import features
+
+
+def uniform_temporal_subsample_video(video: torch.Tensor, num_samples: int, temporal_dim: int = -4) -> torch.Tensor:
+    # Reference: https://github.com/facebookresearch/pytorchvideo/blob/a0a131e/pytorchvideo/transforms/functional.py#L19
+    t_max = video.shape[temporal_dim] - 1
+    indices = torch.linspace(0, t_max, num_samples, device=video.device).long()
```

Comment:
Can we do one of two things here?

1. Rename this to `num_frames` since that is the term we are using throughout the test suite and is more descriptive.
2. (preferred) Merge this into the line below since the variable is only used once after we removed the `.clamp_()` call

```suggestion
    indices = torch.linspace(0, video.shape[temporal_dim] - 1, num_samples, device=video.device).long()
```

Reply:
Can we pass dtype directly into linspace method ?

Reply:
Unfortunately no. See https://github.com/pytorch/vision/pull/6812#discussion_r1002437589